src/components/PageWizard/_page-wizard.scss

```diff
@@ -0,0 +1,38 @@
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~carbon-components/scss/globals/scss/typography';
+
+.page-wizard {
+  display: flex;
+  &--progress {
+    padding-right: $spacing-05;
+    flex: 0 0 12rem;
+    .bx--progress--vertical {
```

Comment:
we should use the `.#{$prefix}--progress--veritcal` since the prefix is configurable and wold cause this selector to fail if changed.

Reply:
thanks, good catch